doc/users/plotting/colormaps/lightness.py

```diff
@@ -79,10 +61,22 @@
 
             # Plot colormap L values
             # Do separately for each category so each plot can be pretty
-            # to make scatter markers change color along plot: http://stackoverflow.com/questions/8202605/matplotlib-scatterplot-colour-as-a-function-of-a-third-variable
-            if cmap_category=='Sequential':
+            # to make scatter markers change color along plot:
+            # http://stackoverflow.com/questions/8202605/matplotlib-scatterplot-colour-as-a-function-of-a-third-variable
```

Comment:
It's (apparently) a little known secret that stackoverflow links can be shortened to http://stackoverflow.com/q/8202605